up42/catalog.py

```diff
@@ -18,28 +18,21 @@
 # TODO: Midterm add catalog results class? Scenes() etc. that also as feedback to workflow input.
 # Scenes() would be dataframe with quicklook preview images in it.
 
-
-blocks_default = [
-    "oneatlas-pleiades-fullscene",
-    "oneatlas-pleiades-aoiclipped",
-    "oneatlas-spot-fullscene",
-    "oneatlas-spot-aoiclipped",
-    "sobloo-sentinel1-l1c-grd-full",
-    "sobloo-sentinel1-l1c-grd-aoiclipped",
-    "sobloo-sentinel1-l1c-slc-full",
-    "sobloo-sentinel2-lic-msi-full",
-    "sobloo-sentinel2-lic-msi-aoiclipped",
-    "sobloo-sentinel3-full",
-    "sobloo-sentinel5-preview-full",
-]
-supported_sensors = [
-    "pleiades",
-    "spot",
-    "sentinel1",
-    "sentinel2",
-    "sentinel3",
-    "sentinel5p",
-]
+supported_sensors_blocks = {
+    "pleiades": ["oneatlas-pleiades-fullscene", "oneatlas-pleiades-aoiclipped",],
+    "spot": ["oneatlas-spot-fullscene", "oneatlas-spot-aoiclipped",],
+    "sentinel1": [
+        "sobloo-sentinel1-l1c-grd-full",
+        "sobloo-sentinel1-l1c-grd-aoiclipped",
+        "sobloo-sentinel1-l1c-slc-full",
+    ],
+    "sentinel2": [
+        "sobloo-sentinel2-lic-msi-full",
+        "sobloo-sentinel2-lic-msi-aoiclipped",
+    ],
+    "sentinel3": ["sobloo-sentinel3-full"],
+    "sentinel5p": ["sobloo-sentinel5-preview-full",],
+}
```

Comment:
Nice! Would be great if we could fetch this info from some endpoint on the backend (maybe specs)?

Reply:
Good idea, added to feature roadmap.